en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -56,8 +50,9 @@ Follow this procedure to test the API revision that you deployed:
 2. Click the `GET` resource you added to expand it.
 
     ![API GET resource](../assets/img/apis/api-resource.png){.cInlineImage-full}
-
-3. Click **Try it out** and then click **Execute**. You can see `{"hello": "world"}` as the response from the API.
+    
+3. Click **GET TEST KEY** to obtain an access token.
```

Comment:
We better move this step to step number 2 as it appears at the top of the page. Then we can ask to do Click the `GET` resource you added to expand it.` as the third step.

Reply:
done